packages-proprietary/react-icons/src/HandWithEuroCoin.tsx

```diff
@@ -5,4 +5,7 @@ const SvgHandWithEuroCoin = (props: SVGProps<SVGSVGElement>) => (
     <path d="M20 21c1.076 0 2 .454 2 2s-1.112 2-2 2h-5q-2 0-2 2h8c.62 0 .817-.228.965-.342l6.823-5.244a2 2 0 1 1 2.437 3.172l-7.187 5.524c-.6.46-1.43.86-2.729.869a4 4 0 0 1-.18.006H0V25h3.987l2.566-2.453A6.3 6.3 0 0 1 10.665 21zM16 1a9 9 0 1 1 0 18 9 9 0 0 1 0-18m0 2a7 7 0 1 0 0 14 7 7 0 0 0 0-14m.6 3q.472 0 .955.086t.794.236l-.526 1.35a2.2 2.2 0 0 0-.584-.181 4 4 0 0 0-.692-.065q-.279 0-.547.07a2 2 0 0 0-.499.204 1.7 1.7 0 0 0-.413.338q-.182.204-.278.46h2.841l-.375 1.116H14.51a1 1 0 0 0-.021.188v.386q0 .102.01.187h2.585l-.365 1.105h-1.994q.075.225.236.423.161.199.397.344t.525.225a2.3 2.3 0 0 0 .611.08q.794 0 1.362-.332l.397 1.362a3 3 0 0 1-.799.3 4.6 4.6 0 0 1-1.088.118q-.708 0-1.271-.193a3.2 3.2 0 0 1-.981-.53 3.1 3.1 0 0 1-.703-.8 3.6 3.6 0 0 1-.423-.997H12l.365-1.105h.482a2 2 0 0 1-.01-.193V10q0-.193.032-.386h-.837l.386-1.115h.611q.172-.59.505-1.051.332-.462.798-.783a3.7 3.7 0 0 1 1.04-.493A4.3 4.3 0 0 1 16.602 6z" />
   </svg>
 )
+/**
+ * @deprecated This icon is deprecated and will be removed in the next major release. You can use the "PersonsWithEuroCoin" icon instead.
```

Comment:
Because we alias this export in the barrel file, the deprecated state is persistent but the message is not.

Reply:
@alimpens Seems like a good reason to iron out the difference between these names after all? We decided against putting in the effort in [this ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-1386) because it was ‘for internal use only’, but a clear deprecation message is public.